nix/shell.nix

```
@@ -0,0 +1,26 @@
+{ pkgs, esy ? pkgs.nodePackages.esy, deku, npmPackages }:
+
+pkgs.mkShell {
+  shellHook = deku.preBuild;
+
+  nativeBuildInputs = (with pkgs; [
+    # Make developer life easier
+    ## General tooling
+    docker
+    docker-compose
+    nodejs-12_x
```

```suggestion
    nodejs-12_x
    ligo
```
I think we should have Ligo here too